external/radiation/radiation/radiation_astronomy.py

```diff
@@ -317,8 +317,8 @@ def sol_update(self, jdate, kyear, deltsw, deltim, lsol_chg, me, solar_data):
         if me == 0:
             print(
                 "for cosz calculations: nswr,deltim,deltsw,dtswh =",
-                f"{nswr[0]}, {deltim[0]}, {deltsw[0]}, {dtswh[0]}, anginc, nstp =",
```

Comment:
Should this debugging change be reverted?

Reply:
It's not debugging ... the print statement (and apparently only the print statement) expects that these scalars actually have a singleton dimension, such that it fails if they don't. This is odd behavior that doesn't affect the port itself so seems easiest to just get rid of it.

Reply:
I don't see any changes responding to this comment, but the tests are passing, so I'll resolve it.